main.css

```diff
@@ -0,0 +1,1010 @@
+  --title: 2.9rem;
+  --subtitle: 1.8rem;
+  --icon: 3rem;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+  font-family: "et-line", sans-serif;
+}
+li {
+  list-style: none;
+}
+a {
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: none;
+}
+h1 {
+  font-size: var(--title);
+}
+p {
+  font-size: var(--text);
+}
+.container {
+  min-height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.section-title {
+  font-size: var(--title);
+  font-weight: 600;
+  color: var(--black);
+  margin-bottom: 10px;
+  text-transform: uppercase;
+  letter-spacing: 0.2rem;
+  text-align: center;
+}
+/* HERO SECTION */
+.principal-image {
+  position: relative;
+  width: 100%;
+  height: 800px;
+}
+.principal-image img {
+  position: absolute;
+  z-index: -1;
+  -webkit-transition: opacity 1s ease-in-out;
+  -moz-transition: opacity 1s ease-in-out;
+  -o-transition: opacity 1s ease-in-out;
+  transition: opacity 1s ease-in-out;
+}
+.principal-image img.top {
+  animation-name: imgchange;
+  animation-timing-function: ease-in;
+  animation-iteration-count: infinite;
+  animation-duration: 5s;
+  animation-direction: alternate;
+}
+.principal-image .header-container {
+  margin: 0 auto;
+  margin-bottom: 30px;
+  text-align: center;
+}
+.principal-image .title-page {
+  font-weight: 600;
+  font-size: var(--title);
+  text-align: center;
+  margin-bottom: 5px;
+  color: var(--white);
+}
+.principal-image .description {
+  font-weight: 400;
+  font-size: var(--subtitle);
+  text-align: center;
+  margin-bottom: 40px;
+  color: var(--white);
+}
+.principal-image a {
+  cursor: pointer;
+  font-size: var(--text);
+  margin-right: 25px;
+  border-radius: 4px;
+  width: 100px;
+  height: 45px;
+  padding: 10px 30px;
+  color: var(--white);
+  background-color: transparent;
+  border: 2px solid var(--standarAqua);
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+  margin-top: 30px;
+  transition: 0.3s ease;
+  transition-property: background-color, color;
+  box-sizing: border-box;
+}
+.principal-image .cta-learn {
+  border: var(--white) 2px solid;
+}
+.principal-image .cta-contact {
+  background-color: var(--standarAqua);
+}
+.cta-contact:hover {
+  color: var(--standarAqua);
+  background-color: var(--white);
+  border-color: var(--white);
+}
+.cta-learn:hover {
+  color: var(--standarAqua);
+  background-color: var(--white);
+  border-color: var(--white);
+}
+.header {
+  border-bottom: 1px solid var(--borderwhite);
+}
+
+/* navbar */
+.container-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.container-header .talos {
+  text-decoration: none;
+  font-weight: 400;
+  color: var(--grey);
+  font-size: var(--title);
+  font-weight: 600;
+}
+.talos:hover {
+  color: var(--hoverAqua);
+}
+.menu {
+  -webkit-transition: all ease 0.2s;
+  -o-transition: all ease 0.2s;
+  transition: all ease 0.2s;
```

Comment:
use only the specific attributes you need on transitions

Reply:
this comment has not been solved: "use only the specific attributes you need on transitions"